### ISSUE TITLE
Provide images with icd2 + wifi integration

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#159]
So I think we're getting close to something that works out of the box.

The latest `libicd-network-wpasupplicant` package has a divert that makes dbus auto launch wpa_supplicant when we need it. This means that `/etc/network/interfaces` should no longer have the `wpa-conf` line. Then, we need icd2 to wait until some user dbus is set up, from xsession, iirc, so my current "fix" is to have icd2 not start until `hildon-desktop` is started completely.

I suspect that if I commit the init script, we should be able to produce images where this works out of the box.

**Comment by MerlijnWajer** [comment]
Latest test image has icd2 with the newest plugin, dbus/wpa diversion, and /etc/network/interfaces change, but it doesn't work yet:

```
network type 'WLAN_ADHOC' has no modules defined
network type 'WLAN_INFRA' has no modules defined
could not map any network type to network modules
no module supporting scanning for type 'WLAN_INFRA' found
```

**Comment by MerlijnWajer** [comment]
Running `dpkg-reconfigure icd2-network-wpasupplicant-config` fixed the problem.

@parazyd ?

**Comment by MerlijnWajer** [comment]
This problem is now solved with libicd-network-wpasupplicant 0.1.5
